aries-core/src/js/components/external/stories/Form.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Grommet,
+  Text,
+  TextInput,
+} from 'grommet';
+import { Hpe } from 'grommet-icons';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'Form',
+};
+
+export const Simple = () => {
```

Comment:
this example should include a simple onSubmit for Form.

Reply:
Sounds good -- I just copy and pasted what we have on the design system site example. For all of these storybook examples, I didn't add to the designs but wanted to put a placeholder for when we receive the designs for storybook examples. I can add an onSubmit for the time being though.